From f105654d60f32bca8fd704b7d4fbfb3b078d80b0 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Sun, 11 Aug 2024 15:02:50 +0800
Subject: [PATCH 3/3] fix(Makefile): add -m32 to default compiler flag

cpuburn uses 32-bit x86 assembly which fails to build with 64-bit assemblers.
Use -m32, lest build fails with the following error for each source:

burnP5.S: Assembler messages:
burnP5.S:15: Error: invalid instruction suffix for `push'
burnP5.S:73: Error: invalid instruction suffix for `pop'
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index acf5157..541f7cc 100644
--- a/Makefile
+++ b/Makefile
@@ -1,3 +1,3 @@
 all : burnP5 burnP6 burnK6 burnK7 burnBX burnMMX
 .S:
-	$(CC) -s -nostdlib -o $@ $<
+	$(CC) -m32 -s -nostdlib -o $@ $<
-- 
2.46.0

